Content.Server/Station/Systems/StationJobsSystem.cs

```diff
@@ -75,6 +75,18 @@ private void OnStationInitialized(StationInitializedEvent msg)
             x => x.Key,
             x=> (int?)(x.Value[1] < 0 ? null : x.Value[1]));
 
+        /// JobGroup logic
+        foreach (var (_, group) in stationJobs.JobGroups)
+        {
+            foreach (var member in group.Members)
+            {
+                if (stationJobs.JobList.TryGetValue(member, out _))
+                {
+                    stationJobs.JobList[member] = group.RoundStart;
+                }
+            }
```

Comment:
```suggestion
            foreach (var member in group.Members.Where(member => stationJobs.JobList.ContainsKey(member)))
            {
                stationJobs.JobList[member] = group.RoundStart;
            }
```
Definitely use `Contains` instead of a `TryGetValue` which throws away the `out` result.

Consider using a LINQ expression instead of a nested `if` and `foreach`.

Reply:
Does the same thing, less readable
After checking the other job code, thats like super hard to contribute, wanna make it look as simple as possible